eng/pipelines/templates/pipeline.yml

```diff
@@ -1,6 +1,22 @@
 jobs:
+- job: Initialize_Build_SemanticVersion
+  timeoutInMinutes: 2
+  pool:
+    name: VSEng-MicroBuildVS2019
+    demands:
+      - DotNetFramework
+      - msbuild
+
+  steps:
+  - template: Initialize_Build_SemanticVersion.yml
+
 - job: Initialize_Build
+  dependsOn: Initialize_Build_SemanticVersion
   timeoutInMinutes: 10
+  variables:
+    SemanticVersion: $[dependencies.Initialize_Build_SemanticVersion.outputs['setsemanticversion.SemanticVersion']]
+    BuildRevision: $[counter(format('{0}.{1}', variables['SemanticVersion'], variables['build.definitionname']), 7130)]
```

Comment:
7130 is a seed value? Do we need to move this higher once your PR is ready to merge? Can it be a build variable instead?

Reply:
I see, you will have a followup PR...